### ISSUE TITLE
Add the previous day for the menu

### ISSUE THREAD

**Issue by ayush-patel** [ucladevx/UCLAEats-backend#1]

**Comment by HirdayGupta** [comment]
@keshpr thoughts?

**Comment by keshpr** [comment]
@HirdayGupta I'm assuming this is for our "hourly scrape"? Sure, it's doable.

**Comment by HirdayGupta** [comment]
Yeah, for hourly scrape. Okay, I’m going to put it on hold for now, in favor of more pressing tasks, but we’ll come back to this later. Thanks!
